### ISSUE TITLE
Allow threading/replies to the alert message

### ISSUE THREAD

**Issue by eberleant** [splunk/slack-alerts#38]
This would be an option to add another message as a reply/thread to the original alert message. For example, the main alert message could contain a short summary, while a longer message (eg stacktrace) could be threaded. This prevents the longer message from taking up a lot of space in the Slack channel, while still being easily accessible/viewable without leaving Slack.

I think it would be enough for most people to allow only a single reply to the original message, but  you could also allow people to add any number of replies.

I'm down to make an attempt on this!

**Comment by gonchik** [comment]
I fully agree, as alerts sometimes can be a lot with the same changes. I will be happy to see the same the in the thread/replies instead of dedicated messages.
